hazelcast-spring/src/main/resources/hazelcast-spring-3.9.xsd

```diff
@@ -2427,6 +2427,7 @@
             <xs:element name="ssl" type="ssl" minOccurs="0" maxOccurs="1"/>
             <xs:element name="aws" type="aws-client" minOccurs="0" maxOccurs="1"/>
             <xs:element name="discovery-strategies" type="discovery-strategies" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="outbound-ports" type="outbound-ports" minOccurs="0" maxOccurs="1"/>
```

Comment:
I added the element with the same name that we have on Member, outbound-ports. Since outbound-ports type is already defined, it should be fine. Do you want me change the name ?